spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/async/AsyncDefaultAutoConfiguration.java

```diff
@@ -33,20 +33,35 @@
 import org.springframework.scheduling.annotation.AsyncConfigurerSupport;
 import org.springframework.scheduling.annotation.EnableAsync;
 
+/**
+ * {@link org.springframework.boot.autoconfigure.EnableAutoConfiguration Auto-configuration}
```

Comment:
not sure how helpful these docs are.. ex if someone looks at javadoc on configuration classes or just jump to source. If the latter, it is repetitive with bean declarations and the class hierarchy.